packages/type-utils/src/builtinSymbolLikes.ts

```diff
@@ -69,6 +69,18 @@ export function isReadonlyTypeLike(
     );
   });
 }
+/**
+ * let F = new Function("foo");
+ *      ^ FunctionLike
+ * let I = (callback: Function) => {}
+ *           ^ FunctionLike
+ */
+export function isFunctionLike(program: ts.Program, type: ts.Type): boolean {
```

Comment:
`typescript` package already exports the function with same name `isFunctionLike`. Having two functions with the same name may cause confusion.